amazon_cloud_code_generator/data/tests/integration/targets/s3/tasks/main.yml

```diff
@@ -19,119 +19,114 @@
         state: absent
       register: output
       ignore_errors: true
-    
+
     - name: Create S3 bucket - check_mode
       amazon.cloud.s3_bucket:
         bucket_name: "{{ bucket_name }}"
         state: present
       check_mode: true
       register: output
-    
+
     - assert:
         that:
             - output is success
             - output is changed
-  
+
     - name: Create S3 bucket
       amazon.cloud.s3_bucket:
         bucket_name: "{{ bucket_name }}"
         state: present
       register: output
-    
+
     - assert:
         that:
             - output is success
             - output is changed
-    
+
     - name: Describe S3 bucket
       amazon.cloud.s3_bucket:
         state: describe
         bucket_name: "{{ output.result.identifier }}"
       register: _result
-    
+
     - assert:
         that:
             - _result is success
-    
+
     - name: Create S3 bucket - idempotence
       amazon.cloud.s3_bucket:
         bucket_name: "{{ output.result.identifier }}"
         state: present
       register: _result
-    
+
     - assert:
         that:
             - _result is success
             - _result is not changed
-    
-    - name: Create S3 bucket - idempotence
+
+    - name: Create S3 bucket - check_mode idempotence
       amazon.cloud.s3_bucket:
         bucket_name: "{{ output.result.identifier }}"
         state: present
       check_mode: true
       register: _result
-    
+
     - assert:
         that:
             - _result is success
             - _result is not changed
 
-    - name: Update S3 bucket public access block configuration and tags - check_mode
+    - name: Update S3 bucket public access block configuration (block_public_policy=true) - check_mode
       amazon.cloud.s3_bucket:
         bucket_name: "{{ output.result.identifier }}"
         state: present
         public_access_block_configuration:
           block_public_acls: false
-          block_public_policy: false
+          block_public_policy: true
```

Comment:
Why did you change it to true?

Reply:
I think that was a result of me moving the tagging tasks to the `tagging.yml`, i didnt actually modify any of the other tasks